DEVDOCS-3498: relativize local links (#1389)
bigcommerce/dev-docs: docs/api-docs/catalog/currencies/currencies-overview.md

BigCommerce’s flexible Currency settings assist developers in building Multi-Currency storefronts that empower shoppers and merchants to do business in their currency of choice. Allowing customers to shop and check out in their native currency provides a more consistent and positive shopping experience, and maintaining price expectations throughout the shopping process encourages conversions for merchants.

> This article provides a high-level guide to multi-currency concepts in BigCommerce as well as specific instructions on [adding currencies](#adding-a-currency) via the control panel. For additional details on how currencies are surfaced throughout BigCommerce APIs, user interfaces, and storefront components, see [How Currencies Works](/api-docs/catalog/currencies/how-currencies-work).

Reviewer: ```suggestion
This article provides a high-level guide to multi-currency concepts in BigCommerce as well as specific instructions on [adding currencies](#adding-a-currency) via the control panel. For additional details on how currencies are surfaced throughout BigCommerce APIs, user interfaces, and storefront components, see [How Currencies Works](/api-docs/multi-currency/guide/how-currencies-work).
```
Looks like this one needs a Netlify redirect

> This article provides a high-level guide to multi-currency concepts in BigCommerce as well as specific instructions on [adding currencies](#adding-a-currency) via the control panel. For additional details on how currencies are surfaced throughout BigCommerce APIs, user interfaces, and storefront components, see [How Currencies Works](/api-docs/catalog/currencies/how-currencies-work).

Author: ```suggestion
This article provides a high-level guide to multi-currency concepts in BigCommerce as well as specific instructions on [adding currencies](#adding-a-currency) via the control panel. For additional details on how currencies are surfaced throughout BigCommerce APIs, user interfaces, and storefront components, see [How Currencies Work](/api-docs/multi-currency/guide/how-currencies-work).
```

> This article provides a high-level guide to multi-currency concepts in BigCommerce as well as specific instructions on [adding currencies](#adding-a-currency) via the control panel. For additional details on how currencies are surfaced throughout BigCommerce APIs, user interfaces, and storefront components, see [How Currencies Works](/api-docs/catalog/currencies/how-currencies-work).

Author: correct subject-verb agreement.

<a id="how-currencies-work"></a>

> For details on how currencies are surfaced throughout BigCommerce APIs, user interfaces, and storefront components, see [How Currencies Works](/api-docs/catalog/currencies/how-currencies-work)

Reviewer: ```suggestion
For details on how currencies are surfaced throughout BigCommerce APIs, user interfaces, and storefront components, see [How Currencies Works](/api-docs/multi-currency/guide/how-currencies-work)
```

> For details on how currencies are surfaced throughout BigCommerce APIs, user interfaces, and storefront components, see [How Currencies Works](/api-docs/catalog/currencies/how-currencies-work)

Author: ```suggestion
For details on how currencies are surfaced throughout BigCommerce APIs, user interfaces, and storefront components, see [How Currencies Work](/api-docs/multi-currency/guide/how-currencies-work)
```

## Related resources

### Articles

> - [How Currencies Works](/api-docs/catalog/currencies/how-currencies-work)

Reviewer: ```suggestion
- [How Currencies Works](/api-docs/multi-currency/guide/how-currencies-work)
```

> - [How Currencies Works](/api-docs/catalog/currencies/how-currencies-work)

Author: ```suggestion
- [How Currencies Work](/api-docs/multi-currency/guide/how-currencies-work)
```
